backend/trafficrules/src/main/java/com/coffeejug/trafficrules/dto/UserDto.java

```diff
@@ -19,9 +20,8 @@ public void setUuid(UUID uuid) {
     }
 
     public void setUuid(String uuid) {
-        if (uuid != null) {
-            this.uuid = UUID.fromString(uuid);
-        }
+        Objects.requireNonNull(uuid);
+        this.uuid = UUID.fromString(uuid);
     }

```

Comment:
In your variant we will have NullPointerException in case null object is passed to method.

Reply:
As for me, this is the correct behavior. We should not allow the users to set `null` as value. It's better to fail fast in this case